client/src/redux/reducers/postingReducer.js

```diff
@@ -13,11 +14,12 @@ const reducer = handleActions(
   {
     [actions.applySuccess]: (state, action) => ({
       ...state,
-      errorMessage: action.payload
+      applicationStatus: 'success'
```

Comment:
In order to avoid unnecessary bugs from comparing 'success' to 'Success', wouldn't it make sense to use a boolean value for applicationStatus (and then rename it as applicationSuccess)? Currently it only has two values, as 'failure' is not really used anywhere.

Reply:
Yeah, good point. I'll change it.